src/calculateRentalCost.js

```diff
@@ -3,8 +3,26 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
   // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  let totalPrice = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return (totalPrice -= LONG_TERM_DISCOUNT);
+  }
+
+  if (days >= SHORT_TERM) {
+    return (totalPrice -= SHORT_TERM_DISCOUNT);
+  } else {
+    return totalPrice;
+  }
 }
```

Comment:
DON'T add else after if with return - the code after it won't be executed anyway.

![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/54860641/46e28d41-68b2-4d66-be54-01c0ad664011)